test/fixtures/entry.css

```diff
@@ -30,6 +30,10 @@
   content: '\f0ce';
 }
 
+.custom-content {
+  content: '\f0'; /* ð */
```

Comment:
shouldn't we assert that the font contains this in our tests?

Reply:
I added this to ensure it does not match all "\f0xx" unicodes from font icons.